kernel/packages/decentraland-loader/store/overrides/emptyParcels.ts

```diff
@@ -0,0 +1,43 @@
+import { list } from './emptyParcelsList'
+import { getEntity } from './getEntity'
+import { ConfigState } from '../config/config'
+
+type EmptyName = keyof typeof list
+const emptySceneNames: EmptyName[] = Object.keys(list) as EmptyName[]
+
+function fakeEmptyId(position: string) {
+  return ('Qm' + position + 'm').padEnd(46, '0')
+}
+
+export function getEmptyEntity(coordinates: string, config: ConfigState) {
```

Comment:
Comment: Might not make sense in this PR, but in the near future we should be fetching these empty scenes from the actual content server (e.g. https://peer.decentraland.org/content/entities/scene?pointer=default1 through https://peer.decentraland.org/content/entities/scene?pointer=default12)